### ISSUE TITLE
[java] ControlStatementBraces is missing checkIfStmt property

### ISSUE THREAD

**Issue by huxi** [pmd/pmd#1004]
**Affects PMD Version:** 
6.2.0

**Rule:**
category/java/codestyle.xml/ControlStatementBraces

**Description:**
`ControlStatementBraces` is supposed to replace `IfStmtMustUseBraces` and `IfElseStmtMustUseBraces` but it only has a `checkIfElseStmt` property, not a `checkIfStmt` property.

We currently exclude `IfStmtsMustUseBraces` in our configurations, but not `IfElseStmtMustUseBraces`. The rationale for this is to allow logging guards like the one in the example code below.

The only way to allow those with the `ControlStatementBraces` rule is by setting `checkIfElseStmt` to `false`. This disables more checks than with the previous rules.

**Code Sample demonstrating the issue:**

```java
if(logger.isDebugEnabled()) logger.debug("{}", foo);
```

**Running PMD through:** *Gradle*

**Comment by oowekyala** [comment]
Hi, thanks for the report! That's definitely an oversight, we'll add a new property to cover your use case.
